modules/flashrom

```diff
@@ -27,6 +27,12 @@ endif
 
 flashrom_target := \
 	$(MAKE_JOBS) \
+	PREFIX="$(INSTALL)" \
+	INSTALL="$(INSTALL)" \
+	DESTDIR="$(INSTALL)" \
+	LIBS_BASE="$(INSTALL)" \
```

Comment:
@macpijan: pkg-config is standing in the way. I satrted to dig into the problem, looking at flashrom's Makefile and Makefile.include to find that LIBS_BASE="$(INSTALL)"  needed to be overriden so that neither libusb was search under host's /usr/ but use Heads libusb. Seems like overriding this makes pkg-config find the proper lib and headers of Heads, while still having issues.

Reply:
```
user@heads-tests:~/heads/install/x86$ cat lib/pkgconfig/libusb-1.0.pc 
prefix=/
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: libusb-1.0
Description: C API for USB device access from Linux, Mac OS X, Windows, OpenBSD/NetBSD and Solaris userspace
Version: 1.0.21
Libs: -L${libdir} -lusb-1.0
Libs.private:  -pthread
Cflags: -I${includedir}/libusb-1.0
user@heads-tests:~/heads/install/x86$ cat lib/pkgconfig/libpci.pc 
prefix=/
includedir=//include
libdir=//lib
idsdir=/

Name: libpci
Description: libpci
Version: 3.5.4
Libs: -L${libdir} -lpci
Libs.private: 
Cflags: -I${includedir}

```